app/controllers/your-psp/yourpsp-tasks.service.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+function stripeSetupStageComplete (account, stage) {
+  if (account.connectorGatewayAccountStripeProgress) {
+    return account.connectorGatewayAccountStripeProgress[stage]
+  }
+  return false
+}
+
+function getNewStripeTaskList (targetCredential, account) {
```

Comment:
```suggestion
function getStripeTaskList (activeCredential, account) {
```

Reply:
Renamed getNewStripeTaskList to getStripeTaskList .